### DISCUSSION TITLE
App running in background

### DISCUSSION THREAD

**Discussion by kakol543** [LeagueOfPoro/CapsuleFarmerEvolved#70] [Ideas]
I'm not very good into programing (running this app using PowerShell and python is the most advanced thing I can do XD), but I have a good idea (at least in my mind). It would be nice if you could minimize this app like steam, origin, discord etc (screenshot). Personally, I like to have few things on the taskbar and this poro icon that I see for few hours make me crazy xD. Maybe you know how to implement this function? I did research but I can't find anything or I can't understand what I find.
![screenshot](https://user-images.githubusercontent.com/33991020/217066645-5a3adc5a-64af-414c-8597-6056ecf62ed8.png)

**Comment by okmenhp** [comment]
You can use WInHide

**Reply by 3Ntr0Py07** [reply]
Since this comment is extremely minimal: It is possible to hide the the window with the help of a program called Winhide, available here: https://sourceforge.net/projects/winhide/
You would download a .zip-file containing the executable (when executed you will not see any window) and a license and readme.txt file. 
For Hiding: 
1. Start winhide.exe
2. Press Shift + Alt + Ctrl - a minimalistic window should have opened now
3. Click on the window you want to hide
4. Click on the "Hide Window"-button of the Winhide window - Both windows should disappear now

For unhiding:
1. See Hiding Step 2 if Winhide is hidden
2. Click on "Show Window"-button.

For Exiting Winhide:
- Close via "Leave Winhide"-button
- Close via Task-Manager
